synfig-studio/src/gui/dialogs/vectorizersettings.cpp

```diff
@@ -266,15 +298,15 @@ static Glib::RefPtr<Gtk::Builder> load_interface(const char *filename) {
 	return refBuilder;
 }
 
-VectorizerSettings* VectorizerSettings::create(/*Gtk::Window& parent*/)
+VectorizerSettings * VectorizerSettings::create(etl::handle<synfig::Layer_Bitmap> my_layer_bitmap,
+ etl::handle<studio::Instance> selected_instance,std::unordered_map <std::string,int>& configmap, etl::handle<synfig::Layer> reference_layer)
 {
 	auto refBuilder = load_interface("vectorizer_settings.glade");
 	if (!refBuilder)
 		return nullptr;
 	VectorizerSettings * dialog = nullptr;
-	refBuilder->get_widget_derived("vectorizer_settings", dialog);
+	refBuilder->get_widget_derived("vectorizer_settings", dialog, my_layer_bitmap, selected_instance, configmap, reference_layer);
 //	if (dialog) {
-//		dialog->set_transient_for(parent);
 //	}
```

Comment:
We need to set this dialog as a child to the main window `->set_transient_for(parent)`
Otherwise, nasty GUI behavior creeps up on some system.

`Gdk-Message: 19:23:33.832: Window 0x39cbdb0 is a temporary window without parent, application will not be able to position it on screen.`

Reply:
I noticed dialog_preview also has this code commented out. Should I make an issue for that?

Reply:
Transient property is set in `Dialog_Preview` constructor itself
https://github.com/synfig/synfig/blob/1ffe5c321e0c8c6a13168e6a4f65025f4fbd0537/synfig-studio/src/gui/dialogs/dialog_preview.cpp#L62-L67

And yes, you can open an issue to remove that redundant code statement :)